db/connection.js

```diff
@@ -0,0 +1,4 @@
+var seedData = require("./seeds");
+module.exports = {
```

Comment:
Since we aren't connecting to mongo or a DB, I might think about renaming this file